src/Helper/GitUserInfoFetcher.php

```diff
@@ -36,13 +36,17 @@ public function getUserName () : string
     /**
      * Fetches the users email from git.
      * If the email can not be fetched it will fallback to the composer package name.
+     * In case the package name can also not be fetched it will simply return the empty string
      *
      * @internal
      */
     public function getUserEmail (KernelInterface $kernel) : string
     {
         $composerFile = $kernel->getProjectDir() . "/composer.json";
-        $packageName = \json_decode(\file_get_contents($composerFile), true)["name"];
+        $composerFileContents = \json_decode(\file_get_contents($composerFile), true);
```

Comment:
Both `\json_decode` and `\json_encode` expose a `\JSON_THROW_ON_ERROR` flag, which will throw an exception instead of silently failing, which ist most likely always what you want.

```suggestion
        $composerFileContents = \json_decode(\file_get_contents($composerFile), true);
```